releasenotes/notes/dagcircuit-separable-circuits-142853e69f530a16.yaml

```diff
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    Introduced a new :class:`.DAGCircuit` class method,
+    :meth:`~.separable_circuits`, which returns a list of :class:`.DAGCircuit` objects,
+    one for each set of connected qubits which have no gates connecting them to another
+    set.
+
+    This method  will not return a DAG solely of classical bits.
+
+issues:
+  - |
+    The subcircuits returned from `DAGCircuit.separable_circuits` will not
+    retain the global phase information of the parent :class:`.DAGCircuit`.
+    All subcircuit global phases will be zero.
```

Comment:
I think we can cut this - it's documented non-bug, and there's nothing mathematical we can really do to change how we do things.